MEMBER_TRAVEL_FUND.md

```diff
@@ -119,11 +119,35 @@ previously received a stipend.
 
 ## Reimbursement
 
-Once the request has been approved, you must email travelapprovals@nodejs.org with following information:
-
-1. Provide receipts as attachments stating your name, the participation covered and the total approved for reimbursement.
-2. Fill out and attach the [Expense Report](./expense-report-template.xls?raw=true) and rename as ExpenseReport-Node-EVENTNAME-YOURNAME-YYYYMM.
-3. Sign the form, date the form, and return it as a PDF. The sign line can be found in the lower left hand corner of the above document. The individual tasked with approving the expense will countersign before sending for processing. You may use an e-signature such as docusign or [the 'markup' feature in Preview](https://support.apple.com/guide/preview/fill-out-and-sign-pdf-forms-prvw35725/mac).
+So that we can reimburse you more quickly, we strongly encourage you to use
+[Expensify](https://expensify.com) to organize and submit your receipts.  Please
+see our [guide to creating and configuring a free Expensify
+account](./reimbursement_process.pdf).
+
+Here are the highlights of the process:
+
+* A free Expensify account is sufficient.
```

Comment:
Nits, add a link
[Expensify](https://www.expensify.com/)

Reply:
Perhaps using the suggestion feature of GitHub here would be helpful:

```suggest
* A free [Expensify](https://www.expensify.com/) account is sufficient.
```

Additionally, quick clarifying question: is Expensify open to global registrations? I assume anyone in the US can register but I am not sure about their international registration policies.

Reply:
Yes, to the best of my knowledge it's possible.  But even if they can't (or don't want to), the fallback is the current process.